### ISSUE TITLE
Allow node-sass to be installed without proxy

### ISSUE THREAD

**Issue by taras** [this-dot/enterprise-ui-dev-box#2]
There are a lot of challenges installing node-sass behind firewalls. 

To eliminate pulling noda-sass binaries over the network, I propose that we include the node-sass binaries via [`SASS_BINARY_SITE`](https://github.com/sass/node-sass/blob/78ac14fb1f8ef7d6b231971634a9fc4a0e4f901c/lib/extensions.js#L78-L101) or [`SASS_BINARY_PATH `](https://github.com/sass/node-sass/blob/78ac14fb1f8ef7d6b231971634a9fc4a0e4f901c/lib/extensions.js#L123-L133)

For more information https://github.com/sass/node-sass/issues/1106

**Comment by repocho** [comment]
Yes please, in my organization it's a total nightmare to manage that. I didn't find a solution for Continuous Integration between win, linux and the proxy + firewall, etc...
We currently evaluating using other technology other than Sass :cry:

**Comment by sivakumar-kailasam** [comment]
@repocho Probably the best thing to do here would be to host all the required node-sass versions' bindings in an internal site copying over all assets from https://github.com/sass/node-sass/releases/tag/v4.7.2 & make use of `SASS_BINARY_SITE`. 

That's probably the approach that we're going to suggest as a part of this docs' readme.
